### PR TITLE
Add presubmits for CNAO release-0.42

### PR DESCRIPTION
**Pull request by phoracek**
Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.42.yaml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "automation/check-patch.e2e-lifecycle-k8s.sh"
+    - name: pull-e2e-cluster-network-addons-operator-lifecycle-okd
```
Comment (qinqon) [inline github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.42.yaml:33]: old stuff.

File: github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.42.yaml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "automation/check-patch.e2e-workflow-k8s.sh"
+    - name: pull-e2e-cluster-network-addons-operator-workflow-okd
```
Comment (qinqon) [inline github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.42.yaml:91]: ditto

**Review by qinqon — CHANGES_REQUESTED** [review]
Remove OKD

File: github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.42.yaml (hunk 1 above)
Comment (phoracek) [reply github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.42.yaml]: Fixed

**Pull request merged**